multiple-domain/MultipleDomain.php

```diff
@@ -141,8 +141,13 @@ public function redirect()
 
         $base = !empty($this->domains[$this->domain]) ? $this->domains[$this->domain] : '';
         $base = is_array($base) ? $base['base'] : $base;
-        if (!empty($base) && !empty($_SERVER['REQUEST_URI']) && strpos($_SERVER['REQUEST_URI'], $base) !== 0) {
-            wp_redirect(home_url($base));
+
```

Comment:
I know this is the default formatting on WP world and this is from the beginning of times. But I really don't like these unnecessary spaces everywhere. This plugin follows [PSR-2](https://www.php-fig.org/psr/psr-2/), which is IMO more concise and pretty.